main.tf

```diff
@@ -122,6 +158,123 @@ resource "observe_dataset" "ingress_logs" {
   }
 }
 
+resource "observe_dataset" "nginx_metrics" {
+  count     = var.enable_nginx_ingress_metrics ? 1 : 0
+  workspace = var.workspace.oid
+  name      = format(var.name_format, "Nginx Ingress Metrics")
+  freshness = var.freshness_default
+
+  inputs = {
+    "pod_metrics" = var.pod_metrics.metrics.oid
+  }
+
+  stage {
+    alias = "filtered_metrics"
+
+    pipeline = <<-EOF
+      filter contains(metric, "nginx")
```

Comment:
I think you can use `startswith(metric, "nginx_")`? Slightly more efficient.

Reply:
Changed